src/interfaces.ts

```diff
@@ -1,11 +1,35 @@
+import { CompositionContext } from 'aurelia-templating';
+import { Container } from 'aurelia-dependency-injection';
+import { NavigationInstruction } from './navigation-instruction';
+import { Router } from './router';
+import { NavModel } from './nav-model';
+import { RouterConfiguration } from './router-configuration';
+import { NavigationCommand } from './navigation-commands';
+import { Next } from './pipeline';
+import { IObservable } from './activation';
+
+/**@internal */
+declare module 'aurelia-dependency-injection' {
+  interface Container {
+    getChildRouter?: () => Router;
+  }
+}
+
+/**@internal */
+declare module 'aurelia-templating' {
```

Comment:
I understand why the above dependency was added. Bummer. This is a breaking change probably. We can live with it though.

Reply:
Maybe not ... ? if we use it in this way, the code we deliver won't get affected ? Or we can just retype `CompositionContext` in this lib

Reply:
Retype `CompositionContext` or a part of it only